browser/ui/webui/brave_new_tab_ui.h

```diff
@@ -1,29 +1,37 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
 
 #ifndef BRAVE_BROWSER_UI_WEBUI_BRAVE_NEW_TAB_UI_H_
 #define BRAVE_BROWSER_UI_WEBUI_BRAVE_NEW_TAB_UI_H_
 
 #include <memory>
+#include <string>
 
 #include "brave/browser/ui/webui/basic_ui.h"
 
 class PrefChangeRegistrar;
+class BraveNewTabMessageHandler;
```

Comment:
nit: This can be deleted.

Reply:
Thanks for spotting, this is fixed now. It was also left over from a previous strategy.